docs/querying.md

```diff
@@ -0,0 +1,111 @@
+# Querying
+
+To get the previously ingested logs back from Loki for analysis, you need a
+client that supports LogQL.  
+Grafana will be the first choice for most users,
+nevertheless [LogCLI](logcli.md) represents a viable standalone alternative.
+
+## Clients
+### Grafana
+
+Grafana ships with built-in support for Loki for versions greater than
+[6.0](https://grafana.com/grafana/download).
```

Comment:
Really people need 6.3 and later to get the best experience, 6.3 is in beta but a release is expected any day now so I think it would be safe to update this to 6.3